packages/prisma/schema.prisma

```diff
@@ -30,60 +30,61 @@ enum PeriodType {
 }
 
 model EventType {
-  id                      Int                     @id @default(autoincrement())
+  id                       Int                     @id @default(autoincrement())
```

Comment:
odd change here

Reply:
this whole migration looks scary

Reply:
I added the space so that the formatting for the file would be consistent